src/de/gurkenlabs/litiengine/environment/GameWorld.java

```diff
@@ -37,6 +37,62 @@
  *
  */
 public final class GameWorld implements IUpdateable {
+
+
+
+  //<editor-fold desc="extra camera vars and consts">
+
+  // Note that this code is placed on top for convenience of development.
+  // The code should be moved to more fitting place once the extra camera feature development had ended.
+
+  // The number of cameras to be loaded into the Game World.
+  // This constant value is temporary, final implementation should
+  // load this from a configuration object / file or set using game settings.
+  // The number of cameras should remain constant for the Game World at all times.
```

Comment:
Not sure about this, what if you want another player to join the game via splitscreen? I could see the number of cameras changing then.

Reply:
I just think it makes the engine easier to handle by having it a constant that can be set by game settings.
So a game that is meant to have support for splitscreen local multiplayer simply just initialize the game with 2 cameras, and use the second camera when needed.

Reply:
Making the number of cameras a dynamic thing could result in null pointer errors and out or range errors.
The additional cameras just take up a bit more memory, regardless.